core/src/validator.rs

```diff
@@ -726,7 +726,7 @@ impl Validator {
             max_slots.clone(),
         );
 
-        let poh_config = Arc::new(genesis_config.poh_config.clone());
+        let poh_config = genesis_config.poh_config.clone();
```

Comment:
a change for another PR, but this clone looks to be superfluous. `poh_config` is only passed by reference below

Reply:
I feel like that change fits with the purpose of this PR.. Good catch.